cs/forms.texy

```diff
@@ -20,9 +20,7 @@ První formulář
 Vytvoříme si v naší aplikaci jednoduchý registrační formulář. Přidáme ho do presenteru pomocí tzv. [továrny |presenters#presenter-a-komponenty]:
 
 /--php
-use Nette\Application\UI;
-
-class HomepagePresenter extends UI\Presenter
+class HomepagePresenter extends Nette\Application\UI\Presenter
```

Comment:
Když zrušíš `use Nette\Application\UI`, přestane fungovat `UI\Form` ve zbytku kódu. Nechal bych to tam.